nixos/tests/noto-fonts.nix

```diff
@@ -0,0 +1,44 @@
+import ./make-test-python.nix ({ pkgs, lib, ... }: {
+  name = "noto-fonts";
+  meta = {
+    maintainers = with lib.maintainers; [ nickcao ];
```

Comment:
```suggestion
    maintainers = with lib.maintainers; [ midchildan ];
```

Reply:
I should've mentioned this in a top level comment, but this is actually correct. The tests originate from @NickCao's code in https://github.com/NixOS/nixpkgs/pull/156080#issuecomment-1019016416. The relevant discussion is in https://github.com/NixOS/nixpkgs/pull/156342#discussion_r790330641.

Reply:
I would be happy to co-maintain this, feel free to add yourself.

Reply:
Thanks, added.